src/dawn/engine_dawn.cc

```diff
@@ -37,6 +39,41 @@ namespace {
 // The minimum multiple row pitch observed on Dawn on Metal.  Increase this
 // as needed for other Dawn backends.
 const uint32_t kMinimumImageRowPitch = 256;
+const float kLodMin = 0.0;
+const float kLodMax = 1000.0;
+
+// Helper stuff
+struct Helper {
```

Comment:
Seems like a better name would be DawnPipelineInfo ?

Reply:
How about DawnPipelineHelper? it's not hold any info, basically it's there so a bunch of variables do not go out of scope while we need them.